### ISSUE TITLE
DNS Resolution Failures on MacOS

### ISSUE THREAD

**Issue by cablespaghetti** [keycloak/terraform-provider-keycloak#837]
When using split DNS on MacOS with this module, for example with a VPN, DNS resolution fails due to using the DNS server from /etc/resolv.conf instead of the proper resolver from the VPN. This is related to the way versions of Go prior to 1.20 do DNS resolution on MacOS when cross-compiled.

This is the same problem: https://github.com/petoju/terraform-provider-mysql/issues/66
More information here: https://danp.net/posts/macos-dns-change-in-go-1-20/

TL;DR can you please update to Go 1.20 so my DNS problems go away 🙏 

Thanks!

**Comment by erdebee** [comment]
As a temporary solution, consider using this:
https://github.com/greenboxal/dns-heaven

Have been using it for years while Terraform keeps on being a pain here.

**Comment by giovannibaratta** [comment]
I ran into the same problem with version 4.4.0 of the provider.

**Comment by Jonomir** [comment]
We have the same issue. Please upgrade the go version 🥺

**Comment by Jonomir** [comment]
Will be resolved by https://github.com/mrparkers/terraform-provider-keycloak/pull/954

**Comment by sschu** [comment]
This should be fixed by now. If not, please reopen.
